syntax = "proto3";
package ecommerce;

service ProductInfo {
  rpc addProduct(Product) returns (ProductID) {}
  rpc getProduct(ProductID) returns (Product) {}
}

message Product {
  string id = 1;
  string name = 2;
}

message ProductID {
  string value = 1;
}